apps/src/blockly/addons/contextMenu.ts

```diff
@@ -404,3 +474,35 @@ function unregisterDefaultOptions() {
     GoogleBlockly.ContextMenuRegistry.registry.unregister('workspaceDelete');
   } catch (error) {}
 }
+
+function registerCustomBlockOptions() {
```

Comment:
Could you add a general comment about how the weights work/what they are for? (Or maybe JS Docs-style comments by default in Blockly functions would be useful generally, which would explain the weights on the functions that use them?)

Reply:
Can do!